config/app_config.go

```diff
@@ -0,0 +1,32 @@
+package config
+
+import (
+	_ "github.com/joho/godotenv/autoload"
+	"github.com/spf13/viper"
+	"log"
+	"os"
+	"path"
+)
+
+func init(){
+	rootPath,_ := os.Getwd()
+	viper.AutomaticEnv()
+	viper.AddConfigPath(".")
+	viper.AddConfigPath( path.Join(rootPath,".."))
+	err := viper.ReadInConfig()
```

Comment:
Maybe something like this?

```suggestion
    if err := viper.ReadInConfig(); err != nil {
        if _, ok := err.(viper.ConfigFileNotFoundError); !ok {
            // log.Panic(err)
        }
    }
```

Reply:
True, this makes sense